Packs/Slack/Integrations/Slack/Slack.py

```diff
@@ -718,10 +786,26 @@ def extract_entitlement(entitlement: str, text: str) -> Tuple[str, str, str, str
     return content, guid, incident_id, task_id
 
 
+class SlackLogger:
+    def __init__(self):
+        self.level = logging.DEBUG
```

Comment:
@amshamah419 are we logging by default on debug? is that intended?

Reply:
Not intended. I added logic to handle the debug mode.